UDPSocket.cpp

```diff
@@ -188,6 +188,8 @@ bool CUDPSocket::open(const unsigned int index, const unsigned int af, const std
 		return false;
 	}
 
+	close(index);
```

Comment:
defensive code

Reply:
At least the previous opened socket if any, is closed, and it marked at -1 in the array.